python/mujinwebstackclient/controllerwebclientraw.py

```diff
@@ -12,11 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import traceback
+import asyncio
+import base64
 import os
 import requests
+import threading
+import traceback
+import uuid
+import websockets
```

Comment:
Can you add websockets to `install_requires` for the client? Make sure to pin websockets to the exact version we use.